x-pack/plugins/cases/public/components/header_page/editable_title.tsx

```diff
@@ -92,7 +92,7 @@ const EditableTitleComponent: React.FC<EditableTitleProps> = ({ onSubmit, isLoad
           }}
           inputAriaLabel="Editable title input field"
           heading="h1"
-          size="s"
+          size="l"
```

Comment:
I am not sure about this change.

This PR is about nesting headers. This change appears to be cosmetic only, do we need it?

Reply:
@adcoelho could you please review reported issue. I see 2 parts here which should be addressed

<img width="846" alt="image" src="https://github.com/elastic/kibana/assets/20072247/a90e5802-4591-47d0-aab1-a13606192839">

@1Copenut fyi